lib/iob/calculate.js

```diff
@@ -61,28 +61,26 @@ function iobCalcBiLinear(treatment, time, dia) {
         time = new Date();
     }
 
-    if (treatment.insulin) {
```

Comment:
Why do you think this is unnecessary?  Is it being checked elsewhere?

Reply:
Previously, the first line in the iobCalc function was: `if (!treatment.insulin) return {}`. At commit c2b5382 I flipped that logic around, though, and placed the entire logic of `iobCalc` inside a `if (treatment.insulin) {}` block.

An alternate could be to add a similar condition block inside the `treatments.forEach(funtion(treatment) {}` block in `total.js`, or we could even place a similar condition within each of the `iobCalcBilinear` and `iobCalcExponential`  functions. Seems like the latter would then be duplicating some logic and the former would be a little too far removed from the actual logic for someone to hold in their head that that condition applies while reading the `iobCalc` logic.